### PR TITLE
update support readme

### PR DESCRIPTION
**Pull request by TalZa1810**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6800/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6800

File: packages/wix-style-react/SUPPORT.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ Once you're absolutely sure the issue is ours, please let us know through the ch
...
+If you think you have found a bug, or have a new feature idea- please start by making sure it hasn't already been [reported or fixed](http://wix.to/i0DYB4w).
 You can search through existing issues and pull requests to see if someone has reported a similar issue to yours.
 
 If it does not exist, please follow the following guidelines:
 
-*   Click [here](https://github.com/wix/wix-style-react/issues/new/choose) to choose the relevant issue template.
+*   Click [here](http://wix.to/i0DYB4w) to get the relevant issue template.
 
 *   Please begin the title with `<ComponentName/>` where appropriate, and use a succinct description that helps others find similar issues.
     *   `<FilePicker/>`- `<some feature>` doesn't work ❌ [(Example)](https://github.com/wix/wix-style-react/issues/6202)
```
Comment (TalZa1810) [inline packages/wix-style-react/SUPPORT.md:94]: Should we leave the example links here? WDYT?
@EgoziE

File: packages/wix-style-react/SUPPORT.md [hunk 2]
```diff
@@ -127,9 +127,6 @@ The current status of each Wix Style React version is as follows:
 
 ### Upgrade from older versions
 
-Read and follow our migration guides:
+Read and follow our migration guide:
 
-*   From 7.x to 8.x/9.x read [V8/V9 migration guide](https://github.com/wix/wix-style-react/blob/master/MIGRATION.md)
-*   From 6.x to 7.x read [V7 migration guide](https://github.com/wix/wix-style-react/blob/version\_7.x/docs/migration/v6-v7.md)
```
Comment (TalZa1810) [inline packages/wix-style-react/SUPPORT.md:133]: the urls are not valit anymore. i couldn't find the readme files of the previous versions.

**Pull request merged**